prizy/settings.py

```diff
@@ -163,3 +163,5 @@
 
 MEDIA_ROOT = os.getenv('PRIZY_MEDIA_ROOT', os.path.join(BASE_DIR, 'media'))
 MEDIA_URL = os.getenv('PRIZY_MEDIA_URL', 'media/')
+
+MEETUPCOM_API_KEY = os.getenv('MEETUPCOM_API_KEY')
```

Comment:
Remember to provide a fallback value when using getenv!

`MEETUPCOM_API_KEY = os.getenv('MEETUPCOM_API_KEY', 'fallback_value')`

Reply:
You also need to import `MEETUP_API_KEY` from `django.conf.settings` to use it in `events.views`